lib/build.js

```diff
@@ -11,6 +11,8 @@ const tokenGen = Symbol('tokenGen');
 const uiUri = Symbol('uiUri');
 const ABORT_CODE = 130; // 128 + SIGINT 2 (^C)
 const TEMPORAL_JWT_TIMEOUT = 12 * 60; // 12 hours in minutes
+const PAGINATE_PAGE = 1;
+const PAGINATE_COUNT = 50;
```

Comment:
does this mean, by default `50` steps for a build will be fetched and caller has to handler if there are more ?

Reply:
Yes. But pagination is not working now, this is doing nothing. Even we don't pass it here, it has a default value in the base.